software/control_sw/src/blocks/noisegen.py

```diff
@@ -72,7 +72,7 @@ def get_seed(self, n):
             self._error('Tried to get noise generator seed for n %d > n_noise (%d)' % (n, self.n_noise))
             return
         regname = 'seeds%d' % ((n*self.n_parallel_samples//2) // 4)
-        return self.get_reg_bits(regname, 8 * ((n*self.n_parallel_samples//2) % 4), 8)
+        return self.get_reg_bits(regname, 8 * ((n*self.n_parallel_samples//2) % 4), wdith=8)
```

Comment:
Typo here looks like this wouldn't work?

Reply:
Correct, apologies - will fix 🙄